frontend/src/utils/helpers.js

```diff
@@ -27,6 +27,12 @@ export const getUserInfo = async () => {
   }
 };
 
+export const getLoginStatus = async () => {
+  const response = await getUserInfo();
+  // Return true if data is not null/undefined
+  return !!response?.data?.userInfo;
```

Comment:
Is the use of `!!` to force it to a boolean (true/false)?

Reply:
Probably can do something like `return response?.data?.userInfo ? true : false;` for more clarity

Reply:
To be resolved on next PR